### PR TITLE
feat: Update transaction trace docs

### PR DESCRIPTION
**Pull request by Marty42Martinez**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

https://new-relic.atlassian.net/browse/NR-185982

The Distributed Tracing team recently took ownership of the Transaction trace UI. As part of the modernization of the code base we are updating the docs to reflect features that currently exist in the UI as well as document new features that we have added.

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by homelessbirds** [comment]
Closing and reopening to help with the build.

**Comment by Marty42Martinez** [comment]
> Closing and reopening to help with the build.

@homelessbirds Hmm it is still failing. Is there anything I need to do on my end?

File: src/content/docs/apm/transactions/transaction-traces/transaction-traces-database-queries-page.mdx [hunk 1]
```diff
@@ -16,11 +16,11 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-import apmTransactionTraceDbQuery from 'images/apm_screenshot-crop_transaction-trace-db-query.webp'
-
-import accountsFastCalls from 'images/apm_screenshot-full_fast-queries.webp'
+import apmTransactionTraceDbQueriesView from '/images/apm_screenshot-crop_transaction-traces-database-queries-view.webp'
```
Comment (clarkmcadoo) [inline src/content/docs/apm/transactions/transaction-traces/transaction-traces-database-queries-page.mdx:19]: I think we need to get rid of this leading `/`

**Comment by homelessbirds** [comment]
Closing because this work has been duplicated.

**Pull request closed without merging**